finite_difference/src/kernel_mod.f90

```diff
@@ -31,12 +31,13 @@ module kernel_mod
 integer, public, parameter :: INTERNAL_PTS = 0, &
```

Comment:
Shouldn't POINTWISE, DOFS, etc all have a GO_ as well?

Reply:
Yes, I'm in two minds about this (you'll see my last commit changed this for instance). I was going to do it for everything but then I realised that would mean a lot of changes in PSyclone. I suspect that I (we) should bite the bullet and make it all consistent. What do you think?

Reply:
*grin* Yes, I counted around 134 'pointwise' alone in the gocen1.0 Fortran test files :) Still, I would be strongly in favour of fixing it, and am happy to do the PSyclone fixes (if you can wait for a few days).

Reply:
Yes, I agree (sigh). Everything is GO'd now...